storage/inchi/src/main/java/org/openscience/cdk/io/INChIContentProcessorTool.java

```diff
@@ -45,7 +45,7 @@ public INChIContentProcessorTool() {}
         public IAtomContainer processFormula(IAtomContainer parsedContent, String atomsEncoding) {
             logger.debug("Parsing atom data: ", atomsEncoding);
 
-            Pattern pattern   = Pattern.compile("([A-Z][a-z]?)(\\d+)?(.*)");
+            Pattern pattern   = Pattern.compile("([A-Z][a-z]?)(\\d++)?(.*+)");
```

Comment:
Never seen this extra + syntax before. Got some info for me? Love to read up on this.